update-vanadium-patches.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+gpwd=$(pwd)
+readonly path="./vanadium_patches/"
+readonly url=https://raw.githubusercontent.com/GrapheneOS/Vanadium/main/patches
+patches=() # current patch names
+smallpatches=() # truncated version of the patches for comparison
+vpatches=() # vanadium patches
+svpatches=() # truncated version for comparisons
+
+# gets the current patches
+get_patches() {
+	cd $path
+	patches=(*.patch)
+	for ((i=0; i<${#patches[@]}; i++)); do
+		smallpatches[$i]="${patches[$i]:4}"
+	done
+	cd $gpwd
+}
+
+# gets Vanadium patches by cloning the repository temporarily
+get_vpatches() {
+	cd uvp-tmp/
+	retry=0
+	while [ true ]; do
```

Comment:
how often does this fail to justify a retry?

Reply:
@RKNF404 i meant in practice, how often does `git clone` from github fail? seems like it wouldn't need a retry